package.json

```diff
@@ -34,13 +34,14 @@
         "@types/enzyme-adapter-react-16": "^1.0.3",
         "@types/jest": "^24.0.0",
         "@types/jsdom": "^12.2.0",
-        "@types/lodash": "^4.14.116",
+        "@types/lodash": "4.14.123",
```

Comment:
I'd prefer not to randomly mix caret and non-caret versions; I'm fine with having a discussion about which to use and sending a PR switching the codebase to discuss from, but I'd rather not do it package by package without discussion.

Reply:
Updated